README.md

```diff
@@ -84,9 +75,31 @@ CFLAGS="-I$GETTEXT/include -I$OPENSSL/include" LDFLAGS=-L$GETTEXT/lib ./configur
 
 You can just run `make` as usual after these steps.
 
+5. Install on macOS with Homebrew
+------------------------------
 
-----------
-5. LICENSE
+  `brew install axel`
+
+```shell
+GETTEXT=/usr/local/opt/gettext
+OPENSSL=/usr/local/opt/openssl
+PATH="$GETTEXT/bin:$PATH"
+./autogen.sh -I$GETTEXT/share/aclocal/
+CFLAGS="-I$GETTEXT/include -I$OPENSSL/include" LDFLAGS=-L$GETTEXT/lib ./configure
+```
+
+You can just run `make` as usual after these steps.
+
```

Comment:
Sorry if I wasn't clear.

Please delete these lines from the 'install' section. They only make sense in the 'building' section.

~~~
+```shell
+GETTEXT=/usr/local/opt/gettext
+OPENSSL=/usr/local/opt/openssl
+PATH="$GETTEXT/bin:$PATH"
+./autogen.sh -I$GETTEXT/share/aclocal/
+CFLAGS="-I$GETTEXT/include -I$OPENSSL/include" LDFLAGS=-L$GETTEXT/lib ./configure
+```
+
+You can just run `make` as usual after these steps.
~~~

All this section needs is `brew install axel`, because that's all you need to do to install it.